playbook/workshops/dodge/add_enemy_sprite.md

```diff
@@ -1,5 +1,8 @@
 # Add Enemy Sprite
 
+_Questions, comments and critiques are welcome! Send your feedback
```

Comment:
It's a bit confusing to have these in the middle of  the workshop like this. I might only put them only at 

1) The bottom of all of the README.md
2) The last step of the tutorial